pkg/domain/domain.go

```diff
@@ -1513,6 +1522,12 @@ func (do *Domain) GetSchemaLease() time.Duration {
 	return do.schemaLease
 }
 
+// IsLeaseExpired returns whether lease has expired
+func (do *Domain) IsLeaseExpired() bool {
+	startReloadTime := time.Now()
+	return time.Since(startReloadTime) > do.schemaLease
```

Comment:
this function seems never return true?
because time.Since(time.Now()) is close to zero.

Reply:
you are absolutely right, that's a dumb mistake. but why did the test passed, let me take a look!